WooCommerce/src/main/kotlin/com/woocommerce/android/model/Product.kt

```diff
@@ -541,7 +544,8 @@ fun WCProductModel.toAppModel(): Product {
         },
         groupedProductIds = this.getGroupedProductIdList(),
         crossSellProductIds = this.getCrossSellProductIdList(),
-        upsellProductIds = this.getUpsellProductIdList()
+        upsellProductIds = this.getUpsellProductIdList(),
+        addons = this.addons?.map { it.toAppModel() } ?: emptyList()
```

Comment:
⛏️ Completely 100% nitpick but IMO a nice one 😄 
```suggestion
        addons = this.addons?.map { it.toAppModel() }.orEmpty()
```

Reply:
Definitely a nice one! Applied your suggestion, thanks!